fog-aliyun.gemspec

```diff
@@ -4,24 +4,25 @@ $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 require 'fog/aliyun/version'
 
 Gem::Specification.new do |spec|
-  spec.name          = "fog-aliyun"
+  spec.name          = 'fog-aliyun'
   spec.version       = Fog::Aliyun::VERSION
-  spec.authors       = ["Qinsi Deng, Jianxun Li, Jane Han"]
-  spec.email         = ["dengqs@dtdream.com"]
+  spec.authors       = ['Qinsi Deng, Jianxun Li, Jane Han']
+  spec.email         = ['dengqs@dtdream.com']
 
-  spec.summary       = %q{Fog provider for Aliyun Web Services.}
-  spec.description   = %q{As a FOG provider, fog-aliyun support aliyun OSS/ECS. It will support more aliyun services later.}
-  spec.homepage      = "https://github.com/fog/fog-aliyun"
-  spec.license       = "MIT"
+  spec.summary       = 'Fog provider for Aliyun Web Services.'
+  spec.description   = 'As a FOG provider, fog-aliyun support aliyun OSS/ECS. It will support more aliyun services later.'
+  spec.homepage      = 'https://github.com/fog/fog-aliyun'
+  spec.license       = 'MIT'
 
   spec.files         = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
-  spec.bindir        = "exe"
+  spec.bindir        = 'exe'
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
-  spec.require_paths = ["lib"]
+  spec.require_paths = ['lib']
 
-  spec.add_development_dependency "bundler", "~> 1.10"
-  spec.add_development_dependency "rake", "~> 10.0"
-  spec.add_development_dependency "rspec", "~> 3.3"
+  spec.add_development_dependency 'bundler', '~> 1.10'
+  spec.add_development_dependency 'rake', '~> 10.0'
+  spec.add_development_dependency 'rspec', '~> 3.3'
```

Comment:
The standard in the `fog` community is to use `minitest`.

Can you switch to minitest plz?